cmd/build.go

```diff
@@ -16,11 +16,30 @@ package cmd
 
 import (
 	"path/filepath"
+	"regexp"
+	"strings"
 
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 )
 
+func checkDockerBuildOptions(options []string) error {
+	buildOptionsTest := "(^((-t)|(--tag)|(-f)|(--file))(=?$)|(=.*))"
```

Comment:
I am not sure we should try and match for `=`.
Example: `docker build --file Dockerfile --tag mytag` would result in no matches for `--file Dockerfile --tag mytag`) - or am I confused?

Reply:
I did some testing - and this seems to yield matches for the blacklisted options: `(^((-t )|(--tag )|(-f )|(--file ))(.*))`

Reply:
The reason the code works is the string is split. and each individual piece is tested, not the whole string at once.

Reply:
Sorry - I had missed that.